website/vercel.json

```diff
@@ -184,6 +184,10 @@
     {
       "source": "/blog/open-source-cspm",
       "destination": "/how-to-guides/open-source-cspm"
+    },
+    {
+      "source": "/blog/configuring-cq-with-localstack",
```

Comment:
Do we need this redirect? This wasn't published as a blog if I'm not mistaken so we can put it directly under https://github.com/cloudquery/cloudquery/tree/8a1c59e9e8f5b966b4f08d149699cacdded13472/website/pages/how-to-guides

Reply:
removed